modules/templates/helper.go

```diff
@@ -120,8 +120,9 @@ func NewFuncMap() []template.FuncMap {
 		"DateFmtShort": func(t time.Time) string {
 			return t.Format("Jan 02, 2006")
 		},
-		"SizeFmt": base.FileSize,
-		"List":    List,
+		"SizeFmt":        base.FileSize,
+		"FormatNumberSI": base.FormatNumberSI,
```

Comment:
This would be better name and more consistent with other template function names (DateFmt, SizeFtm etc):
```suggestion
		"CountFmt": base.FormatNumberSI,
```